### PR TITLE
feat(pci.ai-dashboard): datatr-71- users tokens ai dashboard

### PR DESCRIPTION
**Pull request by jperchoc**
DATATR-71

Signed-off-by: Jonathan Perchoc <jonathan.perchoc@ovhcloud.com>

| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/datatr-106-ai-dashboard-base`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? |no
| Tickets          | Fix #DATATR-71
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

## Description

Add the users & tokens page

## Related
Release branch: https://github.com/ovh/manager/tree/release/ai-dashboard
&nbsp;&nbsp;&nbsp;&nbsp;| -[DATATR-106](https://github.com/ovh/manager/pull/8146)
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| - [DATATR-68](https://github.com/ovh/manager/pull/8160)
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| - [DATATR-69](https://github.com/ovh/manager/pull/8172)
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| - [DATATR-70](https://github.com/ovh/manager/pull/8213)
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| - [DATATR-71](https://github.com/ovh/manager/pull/8443)

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/project/ai-dashboard/tabs/users-tokens/create-token/translations/Messages_fr_FR.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+  "pci_ai_dashboard_create_token_header": "Créer un Token AI",
+  "pci_ai_dashboard_create_token_label_selector": "Selecteur de labels (optionnel)",
+  "pci_ai_dashboard_create_token_name": "Nom",
+  "pci_ai_dashboard_create_token_label_selector_validation_error": "Le sélecteur via label doit avoir la forme suivante <code>label=valeur</code>",
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/ai-dashboard/tabs/users-tokens/create-token/translations/Messages_fr_FR.json:5]: **nitpick:** Can we avoid html in translation files?
```suggestion
  "pci_ai_dashboard_create_token_label_selector_validation_error": "Le sélecteur via label doit avoir la forme suivante {{codeOpeningTag}}label=valeur{{codeClosingTag}}",
```

File: packages/manager/modules/pci/src/projects/project/ai-dashboard/tabs/users-tokens/create-token/translations/Messages_fr_FR.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  "pci_ai_dashboard_create_token_role": "Rôle",
+  "pci_ai_dashboard_create_token_role_help": "Choisissez un rôle d'Administrateur ou simplement d'Opérateur (utilisation simple)",
+  "pci_ai_dashboard_create_token_region": "Région",
+  "pci_ai_dashboard_create_token_region_help": "Région pour laquelle le token sera actif",
+  "pci_ai_dashboard_create_token_add_error": "Une erreur est survenue durant la création du token : {{ message }}",
+  "pci_ai_dashboard_create_token_add_success": "Votre token a été généré avec succès, enregistrez-le : <code>{{ token }}</code>",
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/ai-dashboard/tabs/users-tokens/create-token/translations/Messages_fr_FR.json:11]: Same as https://github.com/ovh/manager/pull/8443/files#r1071033706

File: packages/manager/modules/pci/src/projects/project/ai-dashboard/tabs/users-tokens/create-token/translations/Messages_fr_FR.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  "pci_ai_dashboard_create_token_add": "Générer un token",
+  "pci_ai_dashboard_create_token_common_add": "Générer",
+  "pci_ai_dashboard_create_token_common_cancel": "Annuler",
+  "pci_ai_dashboard_create_token_add_info_1": "Les tokens peuvent être assignés à une ressource spécifique (AI Notebook, Job et App) en utilisant les selecteurs de label ou à toutes vos ressources en laissant le champs vide. Vous pouvez également spécifier les privilèges du jeton pour inclure des droits d'administration ou seulement en lecture.",
+  "pci_ai_dashboard_create_token_add_info_2": "Exemple de sélecteurs : <code>label=valeur</code><code>id=uuid</code><code>type=app</code>",
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/ai-dashboard/tabs/users-tokens/create-token/translations/Messages_fr_FR.json:16]: Same as https://github.com/ovh/manager/pull/8443/files#r1071033706

File: packages/manager/modules/pci/src/projects/project/ai-dashboard/tabs/users-tokens/create-token/translations/Messages_fr_FR.json [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+  "pci_ai_dashboard_create_token_add_info_3": "Par défaut, vos applications auront les labels suivants : <code>id</code> et <code>type</code>",
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/ai-dashboard/tabs/users-tokens/create-token/translations/Messages_fr_FR.json:17]: Same as https://github.com/ovh/manager/pull/8443/files#r1071033706

File: packages/manager/modules/pci/src/projects/project/ai-dashboard/tabs/users-tokens/create-user/translations/Messages_fr_FR.json [hunk 5]
```diff
@@ -0,0 +1,11 @@
+{
+  "pci_ai_dashboard_create_user_heading": "Créer un utilisateur AI",
+  "pci_ai_dashboard_create_user_description": "Description",
+  "pci_ai_dashboard_create_user_role": "Rôle",
+  "pci_ai_dashboard_create_user_add": "Ajouter un utilisateur",
+  "pci_ai_dashboard_create_user_cancel": "Annuler",
+  "pci_ai_dashboard_create_user_role_name_ai_training_operator": "AI Platform - Opérateur",
+  "pci_ai_dashboard_create_user_role_name_ai_training_read": "AI Platform - Lecture seule",
+  "pci_ai_dashboard_create_user_success_message": "L'utilisateur {{ username }} a été ajouté avec le mot de passe <code>{{ password }}</code>",
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/ai-dashboard/tabs/users-tokens/create-user/translations/Messages_fr_FR.json:9]: Same as https://github.com/ovh/manager/pull/8443/files#r1071033706

File: packages/manager/modules/pci/src/projects/project/ai-dashboard/tabs/users-tokens/delete-token/delete-token.routing.js [hunk 6]
```diff
@@ -0,0 +1,25 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state(
+    'pci.projects.project.ai-dashboard.users-tokens.delete-token',
+    {
+      url: '/delete-token',
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/ai-dashboard/tabs/users-tokens/delete-token/delete-token.routing.js:5]: **nitpick:** `token` param needs to be included in the URL to have a consistent behaviour on refresh of Delete token page.

**Review by anooparveti — CHANGES_REQUESTED** [review]

File: packages/manager/modules/pci/src/projects/project/ai-dashboard/tabs/users-tokens/delete-token/delete-token.routing.js (hunk 6 above)
Comment (jperchoc) [reply packages/manager/modules/pci/src/projects/project/ai-dashboard/tabs/users-tokens/delete-token/delete-token.routing.js]: Done here: https://github.com/ovh/manager/pull/8443/commits/2c7a7ce3dcf42f27918184c2a4679322dd24117d

File: packages/manager/modules/pci/src/projects/project/ai-dashboard/tabs/users-tokens/users-tokens.html [hunk 7]
```diff
@@ -0,0 +1,150 @@
...
+    <section>
+        <h4 data-translate="pci_ai_dashboard_users_tokens_users_title"></h4>
+        <div class="mb-3">
+            <span
+                data-translate="pci_ai_dashboard_users_tokens_users_description_1"
+            ></span>
+            <br />
+            <span
+                data-translate="pci_ai_dashboard_users_tokens_users_description_2"
+            ></span>
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/ai-dashboard/tabs/users-tokens/users-tokens.html:20]: **nitpick:** `<p>` would be a better choice if you want them in two different lines.

File: packages/manager/modules/pci/src/projects/project/ai-dashboard/tabs/users-tokens/users-tokens.html [hunk 8]
```diff
@@ -0,0 +1,150 @@
...
+    <section>
+        <h4 data-translate="pci_ai_dashboard_users_tokens_tokens_title"></h4>
+        <div class="mb-3">
+            <span
+                data-translate="pci_ai_dashboard_users_tokens_tokens_description_1"
+            ></span>
+            <br />
+            <span
+                data-translate="pci_ai_dashboard_users_tokens_tokens_description_2"
+            ></span>
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/ai-dashboard/tabs/users-tokens/users-tokens.html:56]: Same as https://github.com/ovh/manager/pull/8443/files#r1083948826

File: packages/manager/modules/pci/src/projects/project/ai-dashboard/tabs/users-tokens/users-tokens.html [hunk 9]
```diff
@@ -0,0 +1,150 @@
...
+            ></oui-datagrid-column>
+            <oui-datagrid-column
+                data-title=":: 'pci_ai_dashboard_users_tokens_tokens_table_region' | translate"
+                data-property="spec.region"
+                data-type="string"
+                data-searchable
+                data-filterable
+                data-sortable="asc"
+            >
+                <span data-ng-bind="$row.spec.region"></span>
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/ai-dashboard/tabs/users-tokens/users-tokens.html:83]: **nitpick:** Its redundant. Can be removed.
```suggestion
```

**Review by ClaveauAlex — APPROVED** [review]

File: packages/manager/modules/pci/src/projects/project/ai-dashboard/tabs/users-tokens/users-tokens.html (hunk 7 above)
Comment (jperchoc) [reply packages/manager/modules/pci/src/projects/project/ai-dashboard/tabs/users-tokens/users-tokens.html]: Changed here: https://github.com/ovh/manager/pull/8443/commits/1e785e41932cdb7bbac7de9f5126103c8cb4d899

File: packages/manager/modules/pci/src/projects/project/ai-dashboard/tabs/users-tokens/users-tokens.html (hunk 8 above)
Comment (jperchoc) [reply packages/manager/modules/pci/src/projects/project/ai-dashboard/tabs/users-tokens/users-tokens.html]: Changed here: https://github.com/ovh/manager/pull/8443/commits/1e785e41932cdb7bbac7de9f5126103c8cb4d899

File: packages/manager/modules/pci/src/projects/project/ai-dashboard/tabs/users-tokens/users-tokens.html (hunk 9 above)
Comment (jperchoc) [reply packages/manager/modules/pci/src/projects/project/ai-dashboard/tabs/users-tokens/users-tokens.html]: Yep, removed here: https://github.com/ovh/manager/pull/8443/commits/1e785e41932cdb7bbac7de9f5126103c8cb4d899

**Review by anooparveti — APPROVED** [review]

**Pull request merged**